tests/conftest.py

```diff
@@ -17,10 +17,18 @@ def compare_against_numpy():
 
     def compare(value, expected):
         sigma = 0.01
-        prob = 0.9999
+        prob = 0.99999
```

Comment:
I was looking into this the other day - changing the probability has only a small effect on the tolerance, and wouldn't have prevented most of the failed unit tests (at least, for the ones that I spot checked). Changing the `sigma` value would have a larger impact; however, it also seemed like the relative error had a much smaller variance than the sigma we use here (around 0.001 for the fast spectral density with y), which makes me wonder if there's something genuinely wrong with the function for some inputs. Or if there's something wrong with the logic of the test.

Reply:
`test_fast_spectral_density_with_y` seems to be where it fails most often. Can you try running that one?

Reply:
yup running that now 100 times, close to completing

Reply:
That one ran but iirfilter tests are flaky.....

Reply:
True, and those tests do only one sample, so that doesn't bode well. There's some edge effects with this filter compared with SciPy, right? Maybe more of the edge needs to be sliced off.